src/sentry/onboarding_tasks/backends/organization_onboarding_task.py

```diff
@@ -47,7 +49,16 @@ def try_mark_onboarding_complete(
 
         organization = Organization.objects.get_from_cache(id=organization_id)
         if features.has("organizations:quick-start-updates", organization, actor=user):
-            required_tasks = OrganizationOnboardingTask.NEW_REQUIRED_ONBOARDING_TASKS
+            # We get the first project, because it is the one that will mark the task of creating the first project as completed
+            first_project = Project.objects.filter(organization=organization).first()
+
+            # If the first project exists and can have source maps, we make source maps required to complete the quick start
+            if first_project and first_project.platform in SOURCE_MAPS:
+                required_tasks = (
+                    OrganizationOnboardingTask.NEW_REQUIRED_ONBOARDING_TASKS_WITH_SOURCE_MAPS
+                )
+            else:
+                required_tasks = OrganizationOnboardingTask.NEW_REQUIRED_ONBOARDING_TASKS
```

Comment:
Does it make sense to check for the first project instead of any project that is setup? I think some orgs may setup the first as backend and the second as frontend?

Reply:
hmm yes it makes sense. Thanks for the great feedback!